manageiq-ui-classic.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.executables   = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency "rails", "~> 5.0.0", ">= 5.0.0.1"
+  s.add_dependency "rails", ">= 5.0.0.1", "< 5.1"
```

Comment:
@chrisarcand Should we switch this line (in master) to better match what is in the core repo:  https://github.com/ManageIQ/manageiq/blob/master/Gemfile#L79